### PR TITLE
Add single param to cool-modals + make backup sheet not interact with other modals

### PR DESCRIPTION
**Pull request by osdnk**
https://rainbowhaus.slack.com/archives/CGUDDJYNL/p1601468875059700

### CODE REVIEW

File: src/handlers/walletReadyEvents.js [hunk 1]
```diff
@@ -65,6 +65,7 @@ export const runWalletBackupStatusChecks = () => {
     setTimeout(() => {
       logger.log('showing BackupSheet with existing_user step');
       Navigation.handleAction(Routes.BACKUP_SHEET, {
+        single: true,
```
Comment (brunobar79) [inline src/handlers/walletReadyEvents.js:68]: There are other occurrences where we navigate to `BACKUP_SHEET`.  Do you need to add the `single` param there too?

- `AlreadyBackupView.js`
- `NeedsBackupView.js`
- `redux/data.js`
- `ImportSeedPhraseSheet.js`

**Review by brunobar79 — COMMENTED** [review]
Aside from my comment LGTM.

@skylarbarrera  Could you please test this one?

File: src/handlers/walletReadyEvents.js (hunk 1 above)
Comment (osdnk) [reply src/handlers/walletReadyEvents.js:68]: fixed

Comment (osdnk) [reply src/handlers/walletReadyEvents.js:68]: It's not expected in AlreadyBackupView and ImportSeedPhraseSheet

**Review by skylarbarrera — APPROVED** [review]
testing on sim + physical device, cant reproduce backup sheet issue

**Pull request merged**